tests/test_publication_info.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+from invenio.testsuite import InvenioTestCase, make_test_suite, run_test_suite
+from flask import current_app
+
+
+class PubInfoTests(InvenioTestCase):
```

Comment:
Is there a test case for the cases when there is publication information and conference information?

Where the output should be:

`Published in Nuevo Cim, 23, 99 (Proceedings of Conference name)`

Looking at the code it seemed that in most cases first the publication information will be shown and in the next line the information about the conference. Not sure this case is taken into account.